<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedSettingsWidget</class>
 <widget class="QWidget" name="AdvancedSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>493</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="dialogContentsLayout">
     <item>
      <widget class="QCheckBox" name="downmixAudioCheckBox">
       <property name="text">
        <string>Downmix Audio from 5.1 to 2.1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="disableBlurCheckbox">
       <property name="toolTip">
        <string>Intel HD Graphics display might face issues if blur is enabled. We strongly recommend to leave option as is.</string>
       </property>
       <property name="text">
        <string>Disable blur</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,1">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QCheckBox" name="doubleBufferCheckbox">
         <property name="text">
          <string>Double Buffering</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="nx::vms::client::desktop::HintButton" name="doubleBufferCheckboxHint" native="true">
         <property name="hint" stdset="0">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="useHardwareDecodingCheckbox">
       <property name="text">
        <string>Use Hardware Video Decoding</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,1">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <widget class="QCheckBox" name="autoFpsLimitCheckbox">
         <property name="text">
          <string>Limit Frame Rate to save CPU</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="nx::vms::client::desktop::HintButton" name="autoFpsLimitCheckboxHint" native="true">
         <property name="hint" stdset="0">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="maximumLiveBufferLengthLabelLayout" stretch="0,1,0">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="maximumLiveBufferLengthLabel">
         <property name="text">
          <string>Maximum Live Buffer Length</string>
         </property>
         <property name="buddy">
          <cstring>maximumLiveBufferLengthSlider</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="nx::vms::client::desktop::HintButton" name="maximumLiveBufferLengthLabelHint" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="hint" stdset="0">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <widget class="QSpinBox" name="maximumLiveBufferLengthSpinBox">
         <property name="suffix">
          <string notr="true">&lt; ms&gt;</string>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>200</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="maximumLiveBufferLengthSlider">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="pageStep">
          <number>100</number>
         </property>
         <property name="value">
          <number>200</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="certificateValidationLabel">
       <property name="text">
        <string>Server certificate validation</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="certificateValidationLevelComboBox"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <widget class="QLabel" name="certificateValidationDescriptionLabel">
         <property name="text">
          <string notr="true">&lt; Description of the current server certificate validation level &gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <widget class="QPushButton" name="clearCacheButton">
         <property name="text">
          <string>Clear Local Cache</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resetAllWarningsButton">
         <property name="text">
          <string>Reset All Warnings</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="logsManagementBox">
       <property name="title">
        <string>Logs Management</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>8</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="logsManagementStack">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="logsDefaultPage">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>8</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <property name="spacing">
               <number>8</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="downloadLogsButton">
                <property name="text">
                 <string>Download</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="logsSettingsButton">
                <property name="text">
                 <string>Settings</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="logsDownloadPage">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>8</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QProgressBar" name="logsDownloadProgressBar">
              <property name="value">
               <number>24</number>
              </property>
              <property name="textVisible">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <property name="spacing">
               <number>8</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="cancelLogsDownloadButton">
                <property name="text">
                 <string>Cancel</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="logsDownloadStatusLabel">
                <property name="text">
                 <string notr="true">&lt; Status &gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_8">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="openLogsFolderButton">
                <property name="text">
                 <string>Open Files Folder</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <property name="spacing">
               <number>8</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="logsDownloadDoneButton">
                <property name="text">
                 <string>Done</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_9">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="nx::vms::client::desktop::AlertBar" name="alertBar" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::vms::client::desktop::HintButton</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/hint_button.h</header>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::AlertBar</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/control_bars.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>doubleBufferCheckbox</tabstop>
  <tabstop>autoFpsLimitCheckbox</tabstop>
  <tabstop>maximumLiveBufferLengthSpinBox</tabstop>
  <tabstop>maximumLiveBufferLengthSlider</tabstop>
  <tabstop>clearCacheButton</tabstop>
  <tabstop>resetAllWarningsButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
